charts/nri-bundle-legacy/requirements.yaml

```diff
@@ -32,7 +32,7 @@ dependencies:
   - name: newrelic-logging
     repository: https://newrelic.github.io/helm-charts
     condition: logging.enabled
-    version: 1.10.9
+    version: 1.10.10
```

Comment:
Tests are failing because of this change (also because you did not `helm dependency update`).

You don't have to update the dependencies of the bundle. We have configured renovatebot that will open PRs automatically with the update of the dependencies. Here you have an example: https://github.com/newrelic/helm-charts/pull/820

This change is a blocker, sorry.

Reply:
Reverted the change